### PR TITLE
[#106] Temporarily load Firefox addon

### PR DESCRIPTION
**Pull request by pmeinhardt**
Partially addresses #106.

Note ⚠️ I don't think there's much we can do to get rid of the warning about `innerHTML` at the moment because it is used for Octicons and friends.

---

Always set an addon-id when building tickety-tick for Firefox.
It appears we should always do so since we are using `storage.sync`.

The relevant bits from [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID):

  > Extensions can explicitly set the add-on ID using the `applications`
    key in manifest.json. However, this key is usually optional.

  > Note, though, that some WebExtension APIs use the add-on ID and
    expect it to be the same from one browser session to the next. If
    you use these APIs in Firefox, then you must set the ID explicitly
    using the `applications` key.

  > Some WebExtension APIs use the add-on ID and expect it to be the
    same from one browser session to the next. If you use these APIs,
    then you must set the ID explicitly using the `applications` key.
    This applies to the following APIs: […], `storage.sync`, […]

### CODE REVIEW

**Comment by pmeinhardt** [comment]
There is one thing I'd like to double check with you @klappradla ⚠️ 

Currently the webpack build sets the addon-id to `tickety-tick@bitcrowd.net`. However this was only used for `firefox-local` builds before, not the ones that we submit to Mozilla.

When I open `about:addons` and move on to "Debug add-ons", Firefox tells me the "Extension ID" for the extension installed from the official Mozilla site is `jid1-ynkvezs8Qn2TJA@jetpack`.

I would like to avoid issues with changing the addon-id, i.e. users loosing their stored preferences because of this (since we're using `storage.sync`/`storage.local` for these and they appear to use the addon-id).

Could you please check which "Extension ID" you see in Firefox? Is it the same as for you?

debug add-ons

If it is the same for you, then it might be safer for us to set this one in the manifest.

**Comment by klappradla** [comment]
Looks like it's the same 👀 

screenshot 2019-01-18 at 00 43 42

**Comment by pmeinhardt** [comment]
@klappradla, I've updated the manifest generation to use the official, random addon-id tickety-tick apparently received when we submitted it. I've checked locally with `yarn build:firefox-local` and `yarn open:firefox-local` and everything seems to work.

Curious to learn what Mozilla thinks of this, when you next submit it 🙂

**Comment by pmeinhardt** [comment]
I just noticed we probably do not need the `firefox-local` variant any more if this works.

File: README.md [hunk 1]
```diff
@@ -77,15 +77,9 @@ Navigate to the [chrome://extensions](//chrome://extensions) page, enable "Devel
 
 ### Firefox
 
-Build the extension with:
+If you just want to try out and debug the extension, go to [about:debugging#addons](//about:debugging#addons). Then press "Load Temporary Add-On" and select the built `manifest.json` from the `dist/firefox` extension directory.
```
Comment (klappradla) [inline README.md]: 🎉

**Pull request merged**